xml/hwbp_registers.xml

```diff
@@ -300,7 +302,8 @@ same project unless stated otherwise.
             hypervisor directly.
         ====
 
-        <field name="hcontext" bits="XLEN-1:0" access="WARL" reset="0">
+        <field name="0" bits="XLEN-1:14" access="R" reset="0" />
+        <field name="hcontext" bits="13:0" access="WARL" reset="0">
```

Comment:
I wanted to change it to 14 bits because there is no value in implementing additional bits. None of our triggers look at bits higher than bit 13 in this register. Or did I miss something?

Reply:
You're right.  I forgot that the width of mhvalue basically makes it useless to have anything wider than 14 bits.  I think that we should change the final sentence of the mcontext register's hcontext field to be:
> If the H extension is implemented, it’s recommended to implement no more than 7 bits on RV32.

Because it would now be impossible to implement more than 14 on RV64.